src/Npgsql/NpgsqlRawCopyStream.cs

```diff
@@ -229,7 +229,7 @@ public void Cancel()
                 }
                 catch (PostgresException e)
                 {
-                    if (e.SqlState == "57014") { return; }
+                    if (e.SqlState == PostgresErrorCodes.QueryCanceled) { return; }
```

Comment:
Unrelated nit: Could  you remove brackets around `return` and put it on the next line?
```suggestion
                    if (e.SqlState == PostgresErrorCodes.QueryCanceled)
                        return;
```